tests/Tests.ServiceProvider/Types.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Tests;
+
+public interface IEcho
```

Comment:
In my experience these files grow out of hand pretty fast, you might want to keep an eye on the size of this file.  🙂